DiscordChatExporter.Core/Markdown/Parsing/MarkdownParser.cs

```diff
@@ -160,7 +160,7 @@ internal static partial class MarkdownParser
     // ... or digit followed by enclosing mark
     // (this does not match all emoji in Discord but it's reasonably accurate enough)
     private static readonly IMatcher<MarkdownNode> StandardEmojiNodeMatcher = new RegexMatcher<MarkdownNode>(
-        new Regex("((?:[\\uD83C][\\uDDE6-\\uDDFF]){2}|[\\u2600-\\u26FF]|\\p{Cs}{2}|\\d\\p{Me})", DefaultRegexOptions),
+        new Regex("((?:[\\uD83C][\\uDDE6-\\uDDFF]){2}|[\\u2600-\\u2604]|[\\u260E]|[\\u2611]|[\\u2614-\\u2615]|[\\u2618]|[\\u261D]|[\\u2620]|[\\u2622-\\u2623]|[\\u2626]|[\\u262A]|[\\u262E-\\u262F]|[\\u2638-\\u263A]|[\\u2640]|[\\u2642]|[\\u2648-\\u2653]|[\\u265F-\\u2660]|[\\u2663]|[\\u2665-\\u2666]|[\\u2668]|[\\u267B]|[\\u267E-\\u267F]|[\\u2692-\\u2697]|[\\u2699]|[\\u269B-\\u269C]|[\\u26A0-\\u26A1]|[\\u26A7]|[\\u26AA-\\u26AB]|[\\u26B0-\\u26B1]|[\\u26BD-\\u26BE]|[\\u26C4-\\u26C5]|[\\u26C8]|[\\u26CE-\\u26CF]|[\\u26D1]|[\\u26D3-\\u26D4]|[\\u26E9-\\u26EA]|[\\u26F0-\\u26F5]|[\\u26F7-\\u26FA]|[\\u26FD]|\\p{Cs}{2}|\\d\\p{Me})", DefaultRegexOptions),
```

Comment:
You don't need square brackets if you're matching a single value. Brackets are only used "any of" enumeration.

So you can simplify your regex like this:

```csharp
new Regex("((?:[\\uD83C][\\uDDE6-\\uDDFF]){2}|[\\u2600-\\u2604\\u260E\\u2611\\u2614-\\u2615...");
```

Reply:
Okay, I'll change it and take a look at regular expression formatting in .NET languages.